### ISSUE TITLE
`invokespecial` on non-`private` fails to work.

### ISSUE THREAD

**Issue by XerTheSquirrel** [SquirrelJME/SquirrelJME#119]
Performing `invokespecial` on a non-`private` fails to work and does not fallback to a normal invoke.

```java
invokenonvirtual squirreljme/compilerbug/TestLLIAByteCode/__checkConcurrent()V

; ...

; Does nothing here for this test
.method __checkConcurrent()V
	return
.end method
```

```java
Java Version: 1.8.0_292
Java Over-Layer: Loading /home/circleci/project/emulators/emulator-base/build/lib/main/release/libemulator-base.so...
JNI Sub-Level: Loading Library...
Java Over-Layer: Binding methods...
JNI Sub-Level: Binding Methods...
JNI Sub-Level: Methods are now bound!
Java Over-Layer: Methods bound!
Java Over-Layer: Loading took 30ms
DB: Registering cldc-compact.jar (/home/circleci/project/modules/cldc-compact/build/squirreljme/vm-main-springcoat/cldc-compact.jar)
DB: Registering cldc.jar (/home/circleci/project/modules/cldc/build/squirreljme/vm-main-springcoat/cldc.jar)
DB: Registering collections.jar (/home/circleci/project/modules/collections/build/squirreljme/vm-main-springcoat/collections.jar)
DB: Registering meep-midlet.jar (/home/circleci/project/modules/meep-midlet/build/squirreljme/vm-main-springcoat/meep-midlet.jar)
DB: Registering gcf.jar (/home/circleci/project/modules/gcf/build/squirreljme/vm-main-springcoat/gcf.jar)
DB: Registering io.jar (/home/circleci/project/modules/io/build/squirreljme/vm-main-springcoat/io.jar)
DB: Registering midp-lcdui.jar (/home/circleci/project/modules/midp-lcdui/build/squirreljme/vm-main-springcoat/midp-lcdui.jar)
DB: Registering meep-swm.jar (/home/circleci/project/modules/meep-swm/build/squirreljme/vm-main-springcoat/meep-swm.jar)
DB: Registering zip.jar (/home/circleci/project/modules/zip/build/squirreljme/vm-main-springcoat/zip.jar)
DB: Registering common-vm.jar (/home/circleci/project/modules/common-vm/build/squirreljme/vm-main-springcoat/common-vm.jar)
DB: Registering tool-manifest-writer.jar (/home/circleci/project/modules/tool-manifest-writer/build/squirreljme/vm-main-springcoat/tool-manifest-writer.jar)
DB: Registering tac.jar (/home/circleci/project/modules/tac/build/squirreljme/vm-main-springcoat/tac.jar)
DB: Registering cldc-compact-test.jar (/home/circleci/project/modules/cldc-compact/build/squirreljme/vm-test-springcoat/cldc-compact-test.jar)
Starting virtual machine (in net.multiphasicapps.tac.MainSingleRunner)...
DB: Starting main thread...
TD: Implement newInstance() access checks.
TD: Check permission: ("java.util.PropertyPermission" "squirreljme.compilerbug.testlliabytecode.1")
DB: SET atmod=int:1
DB: SET vdx=int:1
DB: SET next=int:3
DB: About to run test...
****************************
*** EXTERNAL STACK TRACE ***
cc.squirreljme.vm.springcoat.exceptions.SpringFatalException: BK2z
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.__singleStep(SpringThreadWorker.java:3585)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.run(SpringThreadWorker.java:1246)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.runProcessMain(SpringThreadWorker.java:1345)
	at cc.squirreljme.vm.springcoat.MLEThread$14.handle(MLEThread.java:369)
	at cc.squirreljme.vm.springcoat.MLEDispatcher.dispatch(MLEDispatcher.java:182)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.nativeMethod(SpringThreadWorker.java:1095)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.__vmInvokeStatic(SpringThreadWorker.java:3785)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.__singleStep(SpringThreadWorker.java:2959)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.run(SpringThreadWorker.java:1246)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.run(SpringThreadWorker.java:1216)
	at cc.squirreljme.vm.springcoat.SpringMachine.run(SpringMachine.java:558)
	at cc.squirreljme.vm.springcoat.SpringMachine.runVm(SpringMachine.java:578)
	at cc.squirreljme.emulator.vm.VMFactory.main(VMFactory.java:329)
Caused by: cc.squirreljme.vm.springcoat.exceptions.SpringNoSuchMethodException: BK0x squirreljme/compilerbug/__LLIAFields__ __checkConcurrent:()V
	at cc.squirreljme.vm.springcoat.SpringClass.lookupMethod(SpringClass.java:836)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.__vmInvokeSpecial(SpringThreadWorker.java:3717)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.__singleStep(SpringThreadWorker.java:2945)
	... 12 more
	Suppressed: cc.squirreljme.vm.springcoat.exceptions.SpringVirtualMachineException: BK2y squirreljme/compilerbug/TestLLIAByteCode add:(Ljava/lang/Object;)V 2 null 93 @2#INVOKESPECIAL~:[method squirreljme/compilerbug/TestLLIAByteCode::__checkConcurrent()V]
		at cc.squirreljme.vm.springcoat.SpringThreadWorker.__singleStep(SpringThreadWorker.java:3581)
		... 12 more

*** INTERNAL STACK TRACE ***
EXCEPTION SpringThread #1: main
  | IN squirreljme.compilerbug.TestLLIAByteCode
  |- .add:(Ljava/lang/Object;)V @0h (:93 INVOKESPECIAL@2)
  |- .test:()V @0h (:53 INVOKEVIRTUAL@29)
  | IN net.multiphasicapps.tac.TestRunnable (TestRunnable.java)
  |- .__runTest:([Ljava/lang/Object;)Ljava/lang/Object; @0h (:43 INVOKEVIRTUAL@16)
  | IN net.multiphasicapps.tac.__CoreTest__ (__CoreTest__.java)
  |- .runExecution:([Ljava/lang/String;)Lnet/multiphasicapps/tac/TestExecution; @0h (:173 INVOKEVIRTUAL@379)
  | IN net.multiphasicapps.tac.MainSingleRunner (MainSingleRunner.java)
  |- .main:([Ljava/lang/String;)V @0h (:99 INVOKEINTERFACE@204)
  | IN java.lang.__Start__ (__Start__.java)
  |- .__base:()V @0h (:60 INVOKESTATIC@25)
  |- .__main:()V @0h (:117 INVOKESTATIC@5)

****************************
****************************
*** EXTERNAL STACK TRACE ***
cc.squirreljme.vm.springcoat.exceptions.SpringFatalException: BK2z
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.__singleStep(SpringThreadWorker.java:3585)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.run(SpringThreadWorker.java:1246)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.runProcessMain(SpringThreadWorker.java:1345)
	at cc.squirreljme.vm.springcoat.MLEThread$14.handle(MLEThread.java:369)
	at cc.squirreljme.vm.springcoat.MLEDispatcher.dispatch(MLEDispatcher.java:182)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.nativeMethod(SpringThreadWorker.java:1095)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.__vmInvokeStatic(SpringThreadWorker.java:3785)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.__singleStep(SpringThreadWorker.java:2959)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.run(SpringThreadWorker.java:1246)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.run(SpringThreadWorker.java:1216)
	at cc.squirreljme.vm.springcoat.SpringMachine.run(SpringMachine.java:558)
	at cc.squirreljme.vm.springcoat.SpringMachine.runVm(SpringMachine.java:578)
	at cc.squirreljme.emulator.vm.VMFactory.main(VMFactory.java:329)
Caused by: cc.squirreljme.vm.springcoat.exceptions.SpringNoSuchMethodException: BK0x squirreljme/compilerbug/__LLIAFields__ __checkConcurrent:()V
	at cc.squirreljme.vm.springcoat.SpringClass.lookupMethod(SpringClass.java:836)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.__vmInvokeSpecial(SpringThreadWorker.java:3717)
	at cc.squirreljme.vm.springcoat.SpringThreadWorker.__singleStep(SpringThreadWorker.java:2945)
	... 12 more
	Suppressed: cc.squirreljme.vm.springcoat.exceptions.SpringVirtualMachineException: BK2y squirreljme/compilerbug/TestLLIAByteCode add:(Ljava/lang/Object;)V 2 null 93 @2#INVOKESPECIAL~:[method squirreljme/compilerbug/TestLLIAByteCode::__checkConcurrent()V]
		at cc.squirreljme.vm.springcoat.SpringThreadWorker.__singleStep(SpringThreadWorker.java:3581)
		... 12 more

*** INTERNAL STACK TRACE ***
EXCEPTION SpringThread #1: main
  | IN squirreljme.compilerbug.TestLLIAByteCode
  |- .add:(Ljava/lang/Object;)V @0h (:93 INVOKESPECIAL@2)
  |- .test:()V @0h (:53 INVOKEVIRTUAL@29)
  | IN net.multiphasicapps.tac.TestRunnable (TestRunnable.java)
  |- .__runTest:([Ljava/lang/Object;)Ljava/lang/Object; @0h (:43 INVOKEVIRTUAL@16)
  | IN net.multiphasicapps.tac.__CoreTest__ (__CoreTest__.java)
  |- .runExecution:([Ljava/lang/String;)Lnet/multiphasicapps/tac/TestExecution; @0h (:173 INVOKEVIRTUAL@379)
  | IN net.multiphasicapps.tac.MainSingleRunner (MainSingleRunner.java)
  |- .main:([Ljava/lang/String;)V @0h (:99 INVOKEINTERFACE@204)
  | IN java.lang.__Start__ (__Start__.java)
  |- .__base:()V @0h (:60 INVOKESTATIC@25)
  |- .__main:()V @0h (:117 INVOKESTATIC@5)

****************************
```

**Comment by XerTheSquirrel** [comment]
Fixed by #95.

**Comment by XerTheSquirrel** [comment]
This was cherry picked and fixed with <https://squirreljme.cc/info/9a3ffcb437c02d69>.
